libs/protocol-ui/src/components/molecules/CreatableSelect/CreatableSelect.tsx

```diff
@@ -6,6 +6,16 @@ import FormLabel from '../../atoms/FormLabel';
 import HelperText from '../../atoms/HelperText';
 import ErrorMessage from '../../atoms/ErrorMessage';
 import { UseFormReturn } from 'react-hook-form/dist/types/form';
+import {components} from 'react-select';
+
+export function SelectContainer(props: any) {
```

Comment:
Why is this any? Can we import the react-select types and use it as the type for the prop?